ludwig/encoders/text_encoders.py

```diff
@@ -49,6 +49,11 @@
 logger = logging.getLogger(__name__)
 
 
+def _cls_pooled_error_message(encoder: str):
```

Comment:
Let's add a TODO (and can you file an issue to MLF?) to enforce specific options at the schema level.

Reply:
Yup!